elasticsearch/deprecated.md

```diff
@@ -0,0 +1,3 @@
+This image is officially deprecated in favor of [the `elasticsearh` image provided by elastic.co](https://www.elastic.co/guide/en/elasticsearch/reference/5.2/docker.html) which is available to pull via `docker.elastic.co/elasticsearch/elasticsearch:[version]` like `5.2.1`. This image will receive no further updates after 2017-06-20 (June 20, 2017). Please adjust your usage accordingly.
```

Comment:
I think it's safe to link to `current` instead of `5.2`, these top-level links are unlikely to change (and I'll have it in my head that if they do, we will need to submit a PR for the links to be changed).